app/graphql/helios2_schema.rb

```diff
@@ -4,4 +4,6 @@ class Helios2Schema < GraphQL::Schema
   mutation(Types::MutationType)
   query(Types::QueryType)
   subscription(Types::SubscriptionType)
+
+  orphan_types [Types::GuestsWidget, Types::WeatherWidget, Types::TwitterWidget, Types::NumbersWidget]
```

Comment:
this is interesting. I didn't know orphan types were a thing. Is this just a rails idea? I guess because ruby doesn't do any type inference?

Reply:
Not 100% sure, @cbarber probably knows more, but I think it is only a [gql-ruby](https://graphql-ruby.org/type_definitions/interfaces#orphan-types) interface idea: 
> If you add an object type which implements an interface, but that object type doesn't properly appear in your schema, then you need to add that object to the interfaces's orphan_types

Maybe a consequence of not having type inference?